src/ribbon-fill.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="24" height="24" viewBox="0 0 24 24">
+    <path id="ribbon---fill-a" d="M19.4637875,12.6360071 C18.5344379,13.5392276 18.6256822,13.3834959 18.2838364,14.6208432 C18.1144698,15.2341569 17.6213841,15.7132372 16.9895094,15.8778828 C16.0945214,16.110519 15.999554,16.5025024 16,17 L15.999554,22 L11.9996948,19.7881919 L7.99983568,22 L8,17 C8,16.5 7.90419696,16.1099364 7.0098803,15.8773002 C6.37800558,15.7132372 5.88491991,15.2335743 5.71555332,14.6202606 C5.37553844,13.3876907 5.46428043,13.5368389 4.53621252,12.6353663 C4.07370439,12.1861739 3.89310773,11.5310871 4.06247432,10.9178317 C4.40120751,9.68765044 4.40309953,9.86185086 4.06247432,8.62630961 C3.89310773,8.01241335 4.07370439,7.35790916 4.53621252,6.90871677 C5.46556212,6.00543806 5.37431778,6.16116974 5.71555332,4.92382241 C5.88491991,4.31056701 6.37800558,3.83148672 7.0098803,3.66684111 C8.27167669,3.33807423 8.12104723,3.42639792 9.05289918,2.52078877 C9.51540731,2.07212073 10.1890896,1.89611422 10.820354,2.06075984 C12.0884368,2.39069194 11.9096711,2.39127455 13.179646,2.06075984 C13.8108493,1.89611422 14.4845927,2.07153812 14.9471008,2.52078877 C15.8751687,3.42220313 15.7226472,3.33690901 16.9901197,3.66684111 C17.6219944,3.83090411 18.1150801,4.31056701 18.2844467,4.92382241 C18.6244616,6.15639234 18.5350482,6.00666155 19.4637875,6.90871677 C19.9262956,7.35790916 20.1068923,8.01299596 19.9375257,8.62630961 C19.5975108,9.86004477 19.5981822,9.68642696 19.9375257,10.9184143 C20.1068923,11.5323105 19.9262956,12.1867565 19.4637875,12.6360071 Z M12,7 C10.3434416,7 9,8.34298386 9,10 C9,11.6570476 10.3434416,13 12,13 C13.6565441,13 15,11.6570476 15,10 C15,8.34298386 13.6565441,7 12,7 Z"/>
```

Comment:
Do we want the `id` on there still or no?

Reply:
If we leave it there, an Ember app build pipeline would need to strip the id. Otherwise we can't render multiple instances of the icon on the page at once.

Reply:
That's my fault I forgot to clean it up a bit more.